Objects/descrobject.c

```diff
@@ -1586,9 +1586,15 @@ property_descr_get(PyObject *self, PyObject *obj, PyObject *type)
     propertyobject *gs = (propertyobject *)self;
     if (gs->prop_get == NULL) {
         if (gs->prop_name != NULL) {
-            PyErr_Format(PyExc_AttributeError, "unreadable attribute %R", gs->prop_name);
-        } else {
-            PyErr_SetString(PyExc_AttributeError, "unreadable attribute");
+            PyErr_Format(PyExc_AttributeError,
+                         "property %R of object of type %R has no getter",
+                         gs->prop_name,
+                         PyType_GetQualName(Py_TYPE(obj)));
```

Comment:
I think "property %R of object of type %R" is a bit awkward. I'd prefer "property %R of %R objects has no getter".

Reply:
Yeah, there are lots of different colors this bikeshed could be painted. I think the singular "object" is more consistent with other error messages than the plural "objects", but other than that, I like your suggestion. @Alex-Blade what do you think?

Reply:
To be honest, I don't understand what does it mean. Like is it the same as "Property %R that appears in all objects of type %R..."?

Reply:
Tested out it a bit - strongly disagree with plural form, but in general I like it, so pushed "property %R of %R object has no getter"